guide/interactions/buttons.md

```diff
@@ -81,6 +81,15 @@ client.on('interactionCreate', async interaction => {
 });
 ```
 
+::: warning
+If you're using typescript you'll need to specify the type of components your action row holds. This can be done by specifying the generic parameter in `ActionRowBuilder`.
+
+```diff
+- new ActionRowBuilder()
++ new ActionRowBuilder<MessageActionRowComponentBuilder>()
```

Comment:
If you use `MessageActionRowComponentBuilder` it will allow you to pass both `ButtonBuilder`s and `SelectMenuBuilder`s to the action row

```suggestion
+ new ActionRowBuilder<ButtonBuilder>()
```